pkg/config/testdata/containers_default.conf

```diff
@@ -88,6 +88,9 @@ pids_limit = 2048
 # Unit is optional and can be b (bytes), k (kilobytes), m (megabytes), or g (gigabytes). If the unit is omitted, the system uses bytes.
 shm_size = "65536k"
 
+#Umask inside the container
+umask="0002"
```

Comment:
DOH, think this needs to be "0022"

Reply:
oops!  my bad, this is a test.  Not the default settings.  Way too many emails today.